css/style.css

```diff
@@ -49,7 +49,7 @@ body {
   border-radius: 12px;
 }
 
-#tribute-info {
+.tribute-info {
   margin-top: 10px;
   padding: 5px 10px 10px;
 
```

Comment:
on lines 10 and 56 of your CSS, I noticed you commented out code. If this code is serving no purpose then I suggest you delete them. I think that would help make your code a little clean and less packed.😊